### PR TITLE
Release Pacemaker 2.1.3-rc2

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , can you review this? Thanks. The poki/grace commits have already been reviewed.

**Comment by kgaillot** [comment]
Added commits from #2709

**Comment by kgaillot** [comment]
The poki/grace commits made it in via #2676 so rebased and they're gone

File: xml/api/crm_mon-2.15.rng [hunk 1]
```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (clumens) [inline xml/api/crm_mon-2.15.rng:1]: Shouldn't these both be -2.20.rng?

Comment (clumens) [reply xml/api/crm_mon-2.15.rng:1]: Oh wait, I see what's going on now.  Weird, but yeah I guess that's fine.

Comment (kgaillot) [reply xml/api/crm_mon-2.15.rng:1]: Right, it's a correction to a past point, so an exception to the usual

**Pull request merged**